versions/ufo3/lib.plist.md

```diff
@@ -58,6 +58,12 @@ The dictionary may contain glyph names that are not in the font. The dictionary
 </dict>
 ```
 
+#### public.openTypePostUnderlinePosition
+
+This defines an integer position for underline, defined as the top of the underline from the baseline (the [OpenType post table definition]). If present, this value should be used for `underlinePosition` of the OpenType `post` and `CFF2`. If `postscriptUnderlinePosition` is not defined, this value should be used to calculate the `Type 1` and OpenType `CFF` `underlinePosition`; if `postscriptUnderlinePosition` is present, that value must be used for `Type 1` and `CFF`. 
+
+To calculate the `Type 1`/`CFF` `underlinePosition` from this value, subtract half of `postscriptUnderlineThickness` from `public.openTypePostUnderlinePosition`. If this result is a float, round the value down (i.e. 33.5 rounds to 33, and −33.5 rounds to −34).
```

Comment:
I think the they Type1/CFF key is written as "UnderlinePosition" with the initial capital letter

Reply:
> subtract half...

might be better to write the actual formula as pseudo-code to avoid any confusions

> round the value down

curious, why exactly this? E.g. for outlines' point coordinates we usually do https://github.com/fonttools/fonttools/blob/3235454fcd720f270e18ddf00b058df83da63547/Lib/fontTools/misc/roundTools.py#L23

Reply:
Thinking there is that if one rounds, visually the safer thing is for it to be one unit lower than one unit higher; but I doubt it matters too much — wanted to be explicit here so that behavior is predictable.